### ISSUE TITLE
x/build: skip `darwin-386-10_14` builder on release-branch.go1.12

### ISSUE THREAD

**Issue by bcmills** [golang/go#33756]
As far as I can tell, the `darwin-386-10_14` builder was added for #31751 and has been failing for `release-branch.go1.12` ever since.

I doubt we'll want to fix a deprecated architecture on an old Go version, so we should skip the builder in that configuration instead.

CC @golang/osp-team

**Comment by toothrot** [comment]
I can take a look into this.

**Comment by bcmills** [comment]
@dmitshur, I notice that this builder is currently showing up in the build dashboard for `release-branch.go1.12`. I assume that's an expected wart after https://golang.org/cl/199878 (with the fix gated on the `app/appengine` / `cmd/coordinator` cleanup)?

**Comment by dmitshur** [comment]
@bcmills That is correct.

I think it is a low priority enough wart that it can wait a few weeks (compared to the master branch of the main Go repo, I didn't think that could wait as long), but let me know if you think it's more disruptive and should be fixed sooner.
